index.html

```diff
@@ -37,43 +37,49 @@ <h1>About</h1>
     </section>
 
     <section id="services">
-          <div class="flip-container">
```

Comment:
@PJSalter I formatted the document as well, that's why you see changes similar to this one - hope it's not confusing!

Reply:
It's neater now so it's all good, i read through the changes. Thank You